admin/admin.go

```diff
@@ -73,6 +98,21 @@ func (s *AdminServer) Endpoint() string {
 	return fmt.Sprintf("http://127.0.0.1:%d", s.port)
 }
 
+func (s *AdminServer) ConfigAsString() string {
+	var b strings.Builder
+	fmt.Fprintf(&b, "Admin Server: %s\n\n", s.Endpoint())
+	return b.String()
```

Comment:
nit: doesn't need to be a string builder because it's a single interpolation.

Reply:
done✅